src/controllers/local-storage-page-controller.js

```diff
@@ -1,39 +1,75 @@
 angular.module('ToDo').controller('localStoragePageController',
     function ($scope, ctrlConnect, $uibModal, MODAL_ANSWERS, alertBox) {
 
+        $scope.userChoise = 'allTodos';
+
+        $scope.dropTodos = false;
+        $scope.dropComplTodos = false;
+
         $scope.saveInLocStor = function () {
-            if (localStorage.getItem('todos') !== null && localStorage.getItem('todos') !== '[]') {
+            if (localStorage.getItem($scope.userChoise) !== null && localStorage.getItem($scope.userChoise) !== '[[],[]]') {
                 const modalInstance = $uibModal.open({
                     templateUrl: './js/templates/loc-stor-confirm.html',
                     controller: 'saveToLocContr',
                     resolve: {
                         selectedLanguage() {
                             return $scope.selectedLanguage;
                         },
-                        todosMemoryValue() {
-                            return ctrlConnect.getTodos();
+                        memoryValue() {
+                            const arr = [];
```

Comment:
You use this code in several places. May be good idea make some function to simplify code?
```
const arr = [];
arr[0] = ctrlConnect.getTodos();
arr[1] = ctrlConnect.getComplTodos();
return arr;
```